javascript-cont/vscode-and-style.md

```diff
@@ -71,139 +71,260 @@ Hello Naya Spence!
+As programmers, it's easy to fall into whatever habits our first programming language or learning materials encouraged. But small differences among programmers on concerns even as minor as spacing around operators, where to place curly braces, or how to name variables can make code more difficult to read. Each time we encounter a difference in style, we have to stop and think about it. Was this done intentionally? Is there a reason for it? Following a style guide helps us avoid these unnecessary context switches.
 
-Simply put, style guides help us keep code formatted consistently between developers, making for fewer context switches, and hopefully fewer silly mistakes.
+For a small team with only a few programmers, slight differences may go unnoticed. But as teams get bigger and more and more styles mix, code becomes increasingly ugly and unreadable. In the most extreme cases it can even lead to conflicts among team members, with time wasted arguing over the "right" way to do things, and needless churn as individuals try to "fix" each other's code, which itself can lead to the introduction of new bugs.
+
+Simply put, style guides help us keep code formatted consistently between developers, resulting in fewer context switches, less conflict, and hopefully fewer silly mistakes.
 
 Some common style guides include:
 
 - [AirBNB](https://github.com/airbnb/javascript) has a well documented coding standard that is very popular.
 - [Google](https://google.github.io/styleguide/jsguide.html) also has a popular coding style guide.
 
-We will use the linting setting [ESLint Standard](https://eslint.org/docs/rules/) and [ESLint-react](https://github.com/yannickcr/eslint-plugin-react) which have a rather minimal set of linting rules to highlight common JavaScript problems and will continue to apply when we begin working with React.
+We will use the linting setting [ESLint Standard](https://eslint.org/docs/rules/) and [ESLint-React](https://github.com/yannickcr/eslint-plugin-react) (after introducing the React framework), which have a fairly minimal set of linting rules to highlight common JavaScript problems.
+
+### Applying Style Rules to Our Code
+
+Eslint reports violations of a style guide. We can then fix our code to follow our style rules. We encourage actively watching for Eslint errors, which appear both as wavy underlines and in the Problems VS Code tab, and fixing them as we write code. This trains our eyes to see style issue before they arise, while helping us develop a clean, consistent personal coding style.
```

Comment:
It might be nice to show screenshots of the wavy lines and the Problems tab here. I find that students often don't realize there are different tabs to view output in VSCode.

Reply:
Good idea. I'll grab some screenshots.